shared/common-adapters/floating-menu/menu-layout/index.d.ts

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react'
-import {Color} from '../../../styles'
```

Comment:
unneeded import?

Reply:
Ah, that's right. `StylesCrossPlatform` was something I added for `containerStyle` but ended up going another way.